app/models/workarea/pricing/calculators/taxjar_tax_calculator.rb

```diff
@@ -55,17 +54,6 @@ def fallback
           end
           alias_method :handle_timeout_error, :fallback
 
-          # If doing split shipping (different items go to different shipping
```

Comment:
also of note: inheriting from TaxCalculator means applications only need to decorate one `#price_adjustments_for` method.